microshift.sh

```diff
@@ -101,6 +101,8 @@ version = "*"
 EOF
     sed -i 's/redhat/core/g' scripts/image-builder/config/kickstart.ks.template
     sed -i "/--bootproto=dhcp/a\network --hostname=api.${SNC_PRODUCT_NAME}.${BASE_DOMAIN}" scripts/image-builder/config/kickstart.ks.template
+    sed -i 's/clearpart --all --initlabel/clearpart --all --initlabel --disklabel gpt/g' scripts/image-builder/config/kickstart.ks.template
```

Comment:
is `--initlabel` still required?
> `--initlabel`: Initializes the disk label to the default for your architecture
> `--disklabel DISKLABEL` Set the default disklabel to use.

The latter seems to override the former?

Reply:
Yes as per docs it looks like but I did check one of the kickstart file which uses both the option, can try without `initlabel` and see if that works.

- https://dl.acronis.com/u/storage2/html/AcronisStorage_2_installation_pxe_guide_en-US/creating-a-kickstart-file/kickstart-file-example.html